pkg/cli/containers_rm.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"fmt"
+
+	cli "github.com/acorn-io/acorn/pkg/cli/builder"
+	"github.com/acorn-io/acorn/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func NewContainerDelete() *cobra.Command {
+	cmd := cli.Command(&ContainerDelete{}, cobra.Command{
+		Use: "rm [CONTAINER_NAME...]",
+		Example: `
+acorn container rm my-container`,
+		SilenceUsage: true,
+		Short:        "Delete a container",
+	})
+	return cmd
+}
+
+type ContainerDelete struct {
+}
+
+func (a *ContainerDelete) Run(cmd *cobra.Command, args []string) error {
+	client, err := client.Default()
+	if err != nil {
+		return err
+	}
+
+	for _, container := range args {
+		app, err := client.AppDelete(cmd.Context(), container)
```

Comment:
I dont think we should attempting to do an AppDelete here, should we? We should only be deleting containers, not entire apps via this command.